com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -2,99 +2,101 @@
+| **Neutral HDR Range Reduction Mode** | The curve that the Player uses for tone mapping. The options are:<br />- **BT2390**: The default. Defined by the [BT.2390](https://www.itu.int/pub/R-REP-BT.2390) broadcasting recommendations.<br />- **Reinhard**: A simple Tone Mapping operator.<br /><br />This option is only available when you enable **[Additional Properties](More-Options.html)**. |
+| **Hue Shift Amount**                 | The value determines the extent to which your content retains its original hue after you apply HDR settings. When this value is 0, the tonemapper attempts to preserve the hue of your content as much as possible by only tonemapping [luminance](Physical-Light-Units.html). |
+| **Detect Paper White**               | Enable this property if you want HDRP to use the Paper White value that the display communicates to the Unity Engine. In some cases, the value the display communicates may not be accurate. Implement a calibration menu for your application so that users can display your content correctly on displays that communicate inaccurate values. |
+| **- Paper White**                    | The Paper White value of the display. If you do not enable **Detect Paper White**, you must specify a value here. |
+| **Detect Brightness Limits**         | Enable this property if you want HDRP to use the minimum and maximum nit values that the display communicates. In some cases, the value the display communicates may not be accurate. It is best practice to implement a calibration menu for your application to allow for these situations. |
+| **- Min Nits**                       | The minimum brightness value of the display. If you do not enable **Detect Brightness Limits**, you must specify a value here and in **Max Nits**. |
+| **- Max Nits**                       | The maximum brightness value of the display. If you do not enable **Detect Brightness Limits**, you must specify a value here and in **Min Nits**. |
+
+###### Misuse of **Hue Shift Amount**
 
-While Hue-preserving tonemapping (i.e. Hue Shift Amount set to 0) will better preserve the content colors, sometimes content is authored to rely on the hue shifts that high brightness will produce. A typical example can be for example a very bright flame VFX. The image below shows on the left a flame with Hue Shift Amount set to 0 and on the right the same flame with the hue shift preserved (Hue Shift Amount set to 1).
+Creators might author some content with the intention to use **Hue Shift Amount** to produce special effects. In the illustration below, the **Hue Shift Amount** is 0 for the image A and 1 for image B. The flames image B appear more intense because of the hue shift effect. It is preferable not to author content in this way, because settings optimized for special effects can have undesirable effects on other content in the Scene.
```

Comment:
I just caught a typo I missed in Google Docs. 
Remove "the" in phrase
"for the image A"
which should be "for image A"